roles/wazo-c4-aio/templates/default.j2

```diff
@@ -0,0 +1,26 @@
+server {
+    listen 80 default_server;
+    listen [::]:80 default_server;
+    listen 443 ssl http2 default_server;
+    listen [::]:443 ssl http2 default_server;
+    ssl_certificate /usr/share/xivo-certs/server.crt;
+    ssl_certificate_key /usr/share/xivo-certs/server.key;
+
+    location ^~ /api/auth/ {
+        proxy_pass http://127.0.0.1:9497/;
+
+        proxy_set_header    Host                $http_host;
+        proxy_set_header    X-Script-Name       /api/auth;
+        proxy_set_header    X-Forwarded-For     $proxy_add_x_forwarded_for;
+        add_header          X-Powered-By        wazo-auth always;
+    }
+
+    location ^~ /api/router-confd/ {
+        proxy_pass http://127.0.0.1:9600/;
+
+        proxy_set_header    Host                $http_host;
+        proxy_set_header    X-Script-Name       /api/router-confd;
+        proxy_set_header    X-Forwarded-For     $proxy_add_x_forwarded_for;
+        add_header          X-Powered-By        wazo-router-confd always;
+    }
+}
```

Comment:
I'm clearly not fan of this file. I did it to point the issue where this file need to be in the router-confd and for auth also working like the uc engine.